File: 3701_Compute_Alternating_Sum/solution.py
# class Solution:
#     def alternatingSum(self, nums: List[int]) -> int:
#         result = 0
#         for i in range(len(nums)):
#             if i % 2 == 0:
#                 result += nums[i]
#             else:
#                 result -= nums[i]
#         return result

class Solution:
    def alternatingSum(self, nums: List[int]) -> int:
        return sum(nums[::2]) - sum(nums[1::2])

File: 3701_Compute_Alternating_Sum/solution.c
// int alternatingSum(int* nums, int numsSize) {
//     int result = 0;
//     for (int i = 0; i < numsSize; i++) {
//         if (i % 2 == 0) result += nums[i];
//         else result -= nums[i];
//     }
//     return result;
// }

int alternatingSum(int* nums, int numsSize) {
    int result = 0;
    for (int i = 0; i < numsSize; i++) {
        result += (i % 2 == 0) ? nums[i] : -nums[i];
    }
    return result;
}